### PR TITLE
Add sharing link to See Also

### PR DESCRIPTION
**Pull request by AlexJerabek**

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[d0aea96](https://github.com/OfficeDev/office-scripts-docs/commits/d0aea965fb879afddef317233eb25010edceb685)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/overview/excel.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJerabek-patch-1/docs/overview/excel.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C5%5C19%5C918d1c35-f3a4-631b-1edf-a1a2af22eee5%5CPullRequest%5C202005191653562312-113%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=v%2FRb0%2Fb5eHm1nIBugwXK1%2B0vi79hKZRpNw8s8JsMPXk%3D&st=2020-05-19T16%3A50%3A28Z&se=2020-06-19T16%3A55%3A28Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/overview/excel.md [hunk 1]
```diff
@@ -42,6 +42,7 @@ Complete the [Office Scripts in Excel on the web tutorial](../tutorials/excel-tu
 ## See also
```
Comment (ElizabethSamuel-MSFT) [inline docs/overview/excel.md:42]: Update ms.date?

File: docs/overview/excel.md [hunk 2]
```diff
@@ -42,6 +42,7 @@ Complete the [Office Scripts in Excel on the web tutorial](../tutorials/excel-tu
...
 
 - [Scripting fundamentals for Office Script in Excel on the web](../develop/scripting-fundamentals.md)
```
Comment (ElizabethSamuel-MSFT) [inline docs/overview/excel.md:44]:
```suggestion
- [Scripting fundamentals for Office Scripts in Excel on the web](../develop/scripting-fundamentals.md)
```

**Comment by AlexJerabek** [comment]
Thanks @ElizabethSamuel-MSFT. Your feedback has been addressed and will be checked in.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[2f407a7](https://github.com/OfficeDev/office-scripts-docs/commits/2f407a798b1b8db88446332dbec63be233e22d04)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/overview/excel.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJerabek-patch-1/docs/overview/excel.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C5%5C19%5C918d1c35-f3a4-631b-1edf-a1a2af22eee5%5CPullRequest%5C202005191739145867-113%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=bJ%2B5Bd28xvBJM49%2FGfgZptQ0fQZEdp7Y25j%2FlpDve8M%3D&st=2020-05-19T17%3A35%3A47Z&se=2020-06-19T17%3A40%3A47Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**